engine/components-api/src/main/java/com/cloud/ha/HighAvailabilityManager.java

```diff
@@ -16,18 +16,36 @@
 // under the License.
 package com.cloud.ha;
 
-import java.util.List;
+import static java.lang.String.valueOf;
 
 import com.cloud.deploy.DeploymentPlanner;
 import com.cloud.host.HostVO;
 import com.cloud.host.Status;
 import com.cloud.utils.component.Manager;
 import com.cloud.vm.VMInstanceVO;
+import org.apache.cloudstack.framework.config.ConfigKey;
+
+import java.util.List;
 
 /**
  * HighAvailabilityManager checks to make sure the VMs are running fine.
  */
 public interface HighAvailabilityManager extends Manager {
+
+    ConfigKey<Integer> TimeBetweenCleanup = new ConfigKey<>("Advanced", Integer.class,
+        "time.between.cleanup", valueOf(3600 * 24), "Time in milliseconds to wait before the cleanup thread runs.",
```

Comment:
it should be "seconds" in the description

Reply:
ah, yes right thanks!